licenses/license-engine.sh

```diff
@@ -13,6 +13,8 @@ ALLOWED_LICENSES=(
 	'Apache-2.0'
 	'Apache License, Version 2.0'
 	'Apache*'
+	'Artistic-2.0'
```

Comment:
Seems like this is used by `npm` already, which we obviously use and also distribute (via docker images) extensively, so this should be fine.

https://en.wikipedia.org/wiki/Artistic_License